src/main/kotlin/network/warzone/mars/player/achievements/models/AchievementParent.kt

```diff
@@ -0,0 +1,201 @@
+package network.warzone.mars.player.achievements.models
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed class AchievementParent {
```

Comment:
In the future we should change this to not hardcode the categorization/descriptions of achievements

Reply:
I'm also not sure if I like the name `AchievementParent` here, it sounds more like an `AchievementMetadata` maybe? What do you think?